py21cmsense/observation.py

```diff
@@ -77,6 +84,12 @@ class Observation:
         6 * un.hour,
         validator=(tp.vld_physical_type("time"), ut.between(0 * un.hour, 24 * un.hour)),
     )
+    track: tp.Time | None = attr.ib(
+        None,
+        validator=attr.validators.optional(
+            [tp.vld_physical_type("time"), ut.between(0, 24 * un.hour)]
+        ),
+    )
     obs_duration: tp.Time = attr.ib(
         validator=(tp.vld_physical_type("time"), ut.between(0, 24 * un.hour)),
     )
```

Comment:
Is it possible to actually make these alias? Or just have it so that if you assign to track it sets obs? but if you do both it takes the obs_duration as a preference? is this too much?

Reply:
I don't think it's possible to just make it an alias in any clean way. I think the way that I've done it is as close to an alias as I can manage... i.e. if you set `track`, `obs_duration` is set to `track`. The code never uses the `track` attribute at all, except to set `obs_duration`. Since you can't in-place update `track`, it's not like we have to worry about keeping them in sync.

Reply:
That makes sense to me. Fair enough.